bin/ghe-restore

```diff
@@ -178,6 +178,9 @@ if [ "$GHE_VERSION_MAJOR" -ge 2 ]; then
 fi
 
 echo "Restoring Git repositories ..."
+# Remove temporary 2.2 storage migration directory if it exists
+echo "if [ -d /data/user/repositories-nw-backup ]; then sudo rm -rf /data/user/repositories-nw-backup; fi" |
```

Comment:
Actually, let's leave it. The `sudo` will cause problems on GHE < 2.0 and I don't think it's worth wrapping in a version check.